packages/strapi-builder/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "strapi-builder",
+  "private": true,
+  "version": "0.1.0",
+  "description": "Strapi object builder for internal use only",
+  "main": "index.ts",
+  "scripts": {
+    "test": "echo \"no tests yet\""
+  },
+  "dependencies": {
+    "lodash": "^4.17.12",
```

Comment:
You don't use all deps, could you remove it?

Reply:
removed